src/js/components/Table/Table.js

```diff
@@ -185,7 +200,8 @@ Table.propTypes = {
   className: PropTypes.string,
   mods: PropTypes.string,
   style: PropTypes.object,
-  otherProps: PropTypes.object
+  otherProps: PropTypes.object,
+  maxTableHeight: PropTypes.number
```

Comment:
is this actually a number?  Would I be able to do, `400px` or `40em` or something?

Reply:
Yes this needs to be a string for just that purpose.